lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -58,8 +62,38 @@ def list_jobs(
     service: JobServiceDep,
     skip: int = 0,
     limit: int = 100,
-) -> ListingResponse[JobResponse]:
-    return service.list_jobs(skip, limit)
+) -> ListingResponse[Job]:
```

Comment:
Do we need to update `get_job(service: JobServiceDep, job_id: UUID) -> JobResponse` to `get_job(service: JobServiceDep, job_id: UUID) -> Job` too here?